blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+
+The human element: curation
+
+Production of the initial affiliations database began with two human-generated projects: creating a set of institutional identifiers, and matching affiliation metadata to these identifiers as precisely as possible.  Both of these were done almost entirely by Carolyn Grant.
+
+There are hundreds of thousands of organizations around the world, and in principle any institution or business can be assigned an identifier.  The overwhelming majority of these are not relevant to ADS, so the first task was to establish a list of institutions that match affiliations in our metadata.   The current list stands at about 6600 departments, organizations, and parent institutions in all geographic regions of the world.  Many of these institutions are also tagged with a second identifier, namely the identifier of the parent institution.  For example, the parent institution of the Department of Physics at the University of California at Irvine is “University of California at Irvine”.  For now, our affiliations have two levels only, so in this example, we do not assign “The University of California System” as the parent of UC Irvine, but we do have the System identifier in cases of ambiguity (e.g. Univ. of Cal. with no city or other unambiguous identifying information)
+
+The second, and by far most time-consuming, part of this process was the initial classification of millions of individual affiliation strings from the ADS bibliographic database.  This involved both extraction of the author affiliation strings from the database and identification of the institution represented by that string.
+
+As an example, a typical string might be “Physics Dept., UC Irvine” which would correspond to the affiliation of “Department of Physics, University of California -- Irvine” and then assigning this string its corresponding alpha-numeric ID.  The process sounds straightforward, but extraction of all strings relating to “Department”, “Physics”, “University of California”, and “Irvine” can be long and tedious, especially if you’re searching thousands of possible strings.  There are ways to speed that process (e.g. by searching for “Irvine” and “Physics” in all strings, and then looking through just that set of results), but at the beginning it was an entirely human-curated process.  Complicating things is the fact that many authors are affiliated with more than one institution, for example “Astronomy Dept., U. Texas -- Austin, and UNAM-Morelia”.  Publishers -- especially smaller publications and conference proceedings -- rarely use standardized text for affiliations, and even more rarely list multiple affiliations in separate entities in their metadata.  So the curation process also involves splitting those multiple affiliations and then characterizing each affiliation separately.  That process was repeated over and over for the 6600 affiliations that each occasionally had thousands or tens of thousands of different strings.
```

Comment:
* "that each occasionally had thousands or tens of thousands of *non-unique publisher-provided* strings." or something like that